ocs_ci/ocs/ui/deployment_ui.py

```diff
@@ -253,12 +269,18 @@ def install_internal_cluster(self):
         logger.info("Select all worker nodes")
         self.select_checkbox_status(status=True, locator=self.dep_loc["all_nodes"])
 
+        logger.info("Enable Infra Nodes")
```

Comment:
Lines  235 to 240 and 272 to 277 are exactly the same. Please re-use the code.

Reply:
done